### PR TITLE
refactor(guide): Refactor installation guide

### PR DESCRIPTION
**Pull request by Kitzunu**
Revamp the entire thing to be easier for whoever uses it.

Linux & Mac guides has not changed.

Windows guide has been updated to be simpler. (I use windows)

Any deleted data in deleted files has either been outdated or merged into one of the new files.

### CODE REVIEW

**Comment by Kitzunu** [comment]
@Helias @FrancescoBorzi

File: docs/database-installation.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Database Installation
+
+| Installation Guide | |
+| :- | :- |
+| This article is a part of the Installation Guide. You can read it alone or click on the previous link to easily move between the steps. |
+| [<< Step 3: Server Setup](server-setup.md) | [Step 5: Networking >>](networking.md) |
```
Comment (FrancescoBorzi) [inline docs/database-installation.md]: ... and ?

File: docs/linux-server-setup.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Linux Server Setup
+
+| Installation Guide | |
+| :- | :- |
+| This article is a part of the Installation Guide. You can read it alone or click on the previous link to easily move between the steps. |
+| [<< Step 2: Core Installation](core-installation.md) | [Step 4: Database Installation >>](database-installation.md) |
+
+## Extracting DBC, Maps, VMaps & MMaps
+
```
Comment (FrancescoBorzi) [inline docs/linux-server-setup.md:9]: where are download links?

File: docs/macos-server-setup.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# macOS Server Setup
+
+| Installation Guide | |
+| :- | :- |
+| This article is a part of the Installation Guide. You can read it alone or click on the previous link to easily move between the steps. |
+| [<< Step 2: Core Installation](core-installation.md) | [Step 4: Database Installation >>](database-installation.md) |
+
+## Extracting DBC, Maps, VMaps & MMaps
+
+Go to your AzerothCore build directory (e.g. $HOME/build/bin/) and copy the following files to your World of Warcraft binaries directory.
```
Comment (FrancescoBorzi) [inline docs/macos-server-setup.md:10]: isn't this duplicating the linux one?

Comment (Kitzunu) [reply docs/macos-server-setup.md:10]: No, from the page I took this, written by Helias there are minor things different further down.

File: docs/database-installation.md (hunk 1 above)
Comment (Kitzunu) [reply docs/database-installation.md]: and this means?

File: docs/linux-server-setup.md (hunk 2 above)
Comment (Kitzunu) [reply docs/linux-server-setup.md:9]: in server-setup.md the head page before this page

Comment (Kitzunu) [reply docs/linux-server-setup.md:9]: Any mac/linux/windows page all have a head page that you need to go through before you can access any specifics

**Comment by FrancescoBorzi** [comment]
@azerothcore/developers @azerothcore/wiki @azerothcore/testers please review this big change

**Review by Winfidonarleyan — APPROVED** [review]

**Comment by Winfidonarleyan** [comment]
Look good

**Comment by Kitzunu** [comment]
There are some minor things I discussed with Shin that I will update tomorrow

Thanks @Winfidonarleyan

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Helias** [comment]
merge conflict :pensive:

**Comment by Kitzunu** [comment]
Fixed

**Pull request merged**